### PR TITLE
CP-24213: Show CUs as updates

### PR DESCRIPTION
**Pull request by MihaelaStoica**
CUs are available as both updates and upgrades, but they should be presented as updates where possible.

Signed-off-by: Mihaela Stoica <mihaela.stoica@citrix.com>

### CODE REVIEW

File: XenAdmin/Wizards/PatchingWizard/PatchingWizard_PrecheckPage.cs [hunk 1]
```diff
@@ -400,6 +400,8 @@ private void worker_DoWork(object sender, DoWorkEventArgs e)
                         ? Updates.GetUpgradeSequence(pool.Connection, UpdateAlert, ApplyUpdatesToNewVersion && !automatedUpdatesRestricted)
                         : Updates.GetUpgradeSequence(pool.Connection);
 
+                    log.InfoFormat("Minimal patches for {0}: {1}", pool.Name, string.Join(",", us.MinimalPatches.Select(p => p.Name)));
```
Comment (GaborApatiNagy) [inline XenAdmin/Wizards/PatchingWizard/PatchingWizard_PrecheckPage.cs]: I think this should definitely be after the null check on 'us' (in next line). Additionally, a null check for the MinimalPatches is also needed here.

Comment (GaborApatiNagy) [reply XenAdmin/Wizards/PatchingWizard/PatchingWizard_PrecheckPage.cs]: ...or we can just leave it here, but do the null-check inline (to log that case too)

Comment (MihaelaStoica) [reply XenAdmin/Wizards/PatchingWizard/PatchingWizard_PrecheckPage.cs]: Fixed in the new commit

**Comment by GaborApatiNagy** [comment]
Looks good
Merging

**Pull request merged**